config/application.rb

```diff
@@ -42,5 +42,14 @@ class Application < Rails::Application
 
     # Permitted locales available for the application
     config.i18n.available_locales = [:en, :cn]
+
+    config_files = ['secrets.yml']
+
+    def credentials
```

Comment:
I think this is not needed, I'm guessing something was giving you issues, do you remember what it was?

Reply:
Yes. I was trying to encrypt the credentials and read them.
I understood that this method should be overwritten if we use different keys and credentials files.
Something like this:
```def credentials
      if Rails.env.production?
        super
      else
        encrypted(
          "config/credentials.#{Rails.env}.yml.enc",
          key_path: "config/#{Rails.env}.key"
        )
      end
    end
```

I started that path, but later I was happy with the project working/deploying with a general key and file as a first step.

Reply:
So it is not needed if we use default paths, maybe let's do that since I guess there's no value in having those customised?